packages/theme/src/cli/services/dev.ts

```diff
@@ -119,6 +119,7 @@ export async function dev(options: DevOptions) {
 
   renderLinks(options.store, String(options.theme.id), host, port)
   if (options.open) {
+    // eslint-disable-next-line @typescript-eslint/use-unknown-in-catch-callback-variable
```

Comment:
Don't need this one!

Reply:
This doesn't run in CLI but it complains locally - think this is a new change. Trying to update accordingly / fix this for everyone

[Deactivates linting rules that may incorrectly auto-fix · Shopify/cli@05753cf](https://github.com/Shopify/cli/commit/05753cff5b971c838d45b0a21d5bda32dcd99599)